.github/workflows/pricing-list.yml

```diff
@@ -0,0 +1,15 @@
+name: Call Pricing List
```

Comment:
@SAHU-01 will we adding this workflow to all our repos where we needed the pricing feature?